research/css/index.css

```diff
@@ -1,18 +1,15 @@
-/* Space out content a bit */
```

Comment:
Consider using [liquid layout](http://maxdesign.com.au/articles/liquid/) for organizing all CSS files include this. You can deal with any viewport size with `@media`.

Reply:
ok i'll refer to this. thanks !